proposals/evaluation/uc-3-inheritance.md

```diff
@@ -0,0 +1,190 @@
+# Access Inheritance Use Cases
+
+This is part 3 of the [implementation specific use cases comparison](./use-cases.md).
+
+See also: [UCR 2.3 Collection resource inherited access](https://solid.github.io/authorization-panel/authorization-ucr/#uc-inheritance)
+
+
+## Setup
+
+The Weekly status collection is an `ldp:BasicContainer`, which contains other `ldp:BasicContainers`, one for each weekly meeting, into which further documents can be placed. 
+
+```turtle
+# Resource: </weekly-status/>
+<.>
+  a ldp:BasicContainer ;
+  ldp:contains <2021-04-28/>, <2021-05-05/>, <2021-05-12/> .
+```
+
+We have the following hierarchy of resources (shown in more detail in the UCR):
+
+```
+</weekly-status/>
+</weekly-status/2021-04-28/>
+</weekly-status/2021-04-28/report.md>
+</weekly-status/2021-05-05/>
+</weekly-status/2021-05-05/report.md>
+</weekly-status/2021-05-05/diagram.jpg>
+</weekly-status/2021-05-12/>
+```
+
+
+## 1. Read access to a group on a collection of resources
+
+We want to enable read access to all resources contained in `</weekly-status/>` for a group of people (`ex:Alice` & `ex:Bob`).
+
+### ACP
+
+Bob and Alice are part of the agent matcher `</acp/research#m1>`:
+
+```turtle
+# Resource: </acp/research>
+<#m1>
+  a acp:AgentMatcher ;
+  acp:agent ex:Bob, ex:Alice .
+```
+
+The research policy `</acp/research#p1>` gives read access to all agents matched by `</acp/research#m1>`:
+
+```turtle
+# Resource: </acp/research>
+<#p1>
+  a acp:Policy ;
+  acp:anyOf <#m1> ;
+  acp:allow acl:Read .
+```
+
+The access control `</weekly-status/.acp#ac1>` applies to all resources contained by `</weekly-status/>` and via policy `</acp/research#p1>` enables read access for all agents matched by `</acp/research#m1>`:
+
+```turtle
+# Resource: </weekly-status/.acp>
+<#ac1>
+  a acp:AccessControl ;
+  acp:apply </acp/research#p1> ; # applies the policy to the container resource </weekly-status/>
+  acp:applyMembers </acp/research#p1> . # applies the policy to all resources contained by </weekly-status/>
+```
+
+Todo:  what is the ACR for 
+`</weekly-status/2021-04-28/report.md>` and what information is in it?
+   
+### WAC
+
+Bob and Alice are members of the research group `</groups/research#g1>`:
+
+```turtle
+# Resource: </groups/research>
+<#g1>
+  a vcard:Group ;
+  vcard:hasMember ex:Bob ;
+  vcard:hasMember ex:Alice .
+```
+
+The acl enabling read access to all resources contained by `</weekly-status/>` for all members of group `</groups/research#g1>` is:
+
+```turtle
+# Resource: </weekly-status/.acl>
+<#authorization>
+  a acl:Authorization ;
+  acl:agentGroup </groups/research#g1> ;
+  acl:default <.> ;
+  acl:mode acl:Read .
+```
+
+
+## 2. Changing permissions to a subcollection
+
+Bob wants to give Carol read/write access but only to the collection `</weekly-status/2021-04-28/>`.
```

Comment:
Further tweak to @matthieubosquet's text suggestion just above.  Note -- the backticks around the angle-brackets (`< >`) in the text are necessary!
```suggestion
Bob wants to give Carol read/write access to the "2021-04-28" weekly status, and nothing more; that is, to the collection `</weekly-status/2021-04-28/>` and every resource it contains, but not to any other resource in the collection `</weekly-status/>`.
```